src/EEXFormatter.ts

```diff
@@ -122,6 +121,6 @@ export class EEXFormattingEditProvider implements vscode.DocumentFormattingEditP
                     break
             }
             return acc
-        }, acc)
+        }, ["--engine", "eex"])
```

Comment:
Instead of overriding all the settings you could add an option to the config and add `--engine eex` to the acc array inside of the reduce callback.

Reply:
I actually don't override anything here, this is a base value for accumulator that the configs build on in reduce :) I can see this being a bit messy though, any other ideas how to make it look better? I prefer not to add this to the config as setting this value shouldn't really be users responsibility - it should be a default behaviour for this addon (passing this flag to htmlbeautifier).

Reply:
Oh sorry, my fault 😂 still in holiday mode.

Reply:
anyway, for this PR to make sense we need to have the htmlbeautifier merge on the elixir changes and I'm not sure if it will ever happen ;) If it doesn't then maybe I will work on something separate then to be able to share this, as I believe with those changes the formatter works much more reliably now for me for elixir templates